plugins/filetools/plugin.js

```diff
@@ -867,21 +867,18 @@
 		},
 
 		/**
-		 * Property inform if current browser poses native methods used by {@link CKEDITOR.fileTools} to upload files.
+		 * Feature detection indicating whether current browser supports methods essential to send files over XHR request.
 		 *
+		 * @since 4.8.1
 		 * @property {Boolean} isFileUploadSupported
 		 */
 		isFileUploadSupported: ( function() {
-			if ( typeof FileReader === 'function' &&
+			return typeof FileReader === 'function' &&
```

Comment:
Actually `FileReader` is not needed alongside `FormData`. Making XHR request using `FormData` with passed file input should be easy to do. I wonder if we could modify our file loaders accordingly.

Reply:
Because ATM our file loader transforms `Blob` (so binary format ready to be send via xhr) to Data URI, which is probably longer and weights more than binary format. This operation is also completely redundant in case we already have _better_ format.

Reply:
That's true, however it's a separate issue that is beyond scope of this one.